src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -4236,6 +4236,11 @@ namespace HeatBalanceSurfaceManager {
 
             if (Surface(SurfNum).EMSConstructionOverrideON && (Surface(SurfNum).EMSConstructionOverrideValue > 0)) {
 
+                if (Construct(Surface(SurfNum).EMSConstructionOverrideValue).TypeIsWindow) { // okay, allways allow windows
+                    EMSConstructActuatorChecked(Surface(SurfNum).EMSConstructionOverrideValue, SurfNum) = true;
+                    EMSConstructActuatorIsOkay(Surface(SurfNum).EMSConstructionOverrideValue, SurfNum) = true;
+                }
```

Comment:
OK, so this just catches all windows ahead of time and sets the checked/OK flags to true prior to the block under it.  In the original case, I guess the windows would sometimes get to this point with these flags set to false.  Fine with me.